### DISCUSSION TITLE
突然变成免费版，请协助恢复

### DISCUSSION THREAD

**Discussion by TGuby** [getlantern/lantern-forum-cn#250] [Q&A] [answered]
所有设备上的都变成免费版了，你们又没有账户体系，qq邮箱又收不到，简直了
zfb订单号2024022322001359791426573943
充两年版617，请尽快协助恢复

**Comment by MaxMKMax** [comment, marked as answer]
@TGuby 您好。对于造成的不便，我们深表歉意。请问您能提供您注册付费版本的电子邮件吗？

**Reply by TGuby** [reply]
因为没有记录，只能凭印象，大概是1794606794的qq邮箱

**Comment by TGuby** [comment]
***@***.******@***.***，因为没有记录，只能凭印象



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年3月18日 16:37
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: TGuby ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 突然变成免费版，请协助恢复 (Discussion #250)





 
@TGuby 您好。对于造成的不便，我们深表歉意。请问您能提供您注册付费版本的电子邮件吗？

**Reply by MaxMKMax** [reply]
@TGuby 好的. 已经帮您提交上去了. 谢谢您支持蓝灯!

**Reply by TGuby** [reply]
还是没有解决啊，有什么方案？

**Reply by MaxMKMax** [reply]
@TGuby 您好。您是 IOS 还是 Android 用户？今晚我会再次为您跟进。

**Comment by TGuby** [comment]
主要是用pc win11，手机的话安卓有认证过。



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年3月21日 20:40
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: TGuby ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 突然变成免费版，请协助恢复 (Discussion #250)





 
@TGuby 您好。您是 IOS 还是 Android 用户？今晚我会再次为您跟进。

**Reply by MaxMKMax** [reply]
@TGuby 好的。已经通知负责人了。请在线等待他们，因为他们现在离线。他们将在今天午夜再次跟进您的问题。

**Reply by TGuby** [reply]
好像没有任何进展？

**Reply by MaxMKMax** [reply]
@TGuby 您好. 我已经帮您问第三或第四次了. 看来客服那里最近超级忙. 我已经帮您追了刚才.
请您稍等. 不好意思啊. 非常抱歉!

**Comment by MaxMKMax** [comment]
@TGuby 
您好. 支付团队帮您弄好了. 这是您的激活码. 赶紧登陆哦! 
H7X7C-GD27C-2PWX7-9B28H-PCGRB

**Reply by channsguo** [reply]
@Mike777Khoo 你好如果激活码可以解决不能登录的问题也帮我给个激活码谢谢

**Reply by channsguo** [reply]
我的账号是312018325@qq.com

**Reply by MaxMKMax** [reply]
@channsguo 
您好. 在另一个帖子给您发激活码了哦. 请看看.

**Comment by woodballhead** [comment]
我的也是同样的问题，我的账号是wuyiwen798@163.com，麻烦弄个激活码

**Comment by TGuby** [comment]
提示“兑换激活码时出错”，是从专业版激活那边进没错吧？？



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: Mike K ***@***.***&gt;
发送时间: 2024年3月26日 13:51
收件人: getlantern/lantern-forum-cn ***@***.***&gt;
抄送: TGuby ***@***.***&gt;, Mention ***@***.***&gt;
主题: Re: [getlantern/lantern-forum-cn] 突然变成免费版，请协助恢复 (Discussion #250)





 
@TGuby
 您好. 支付团队帮您弄好了. 这是您的激活码. 赶紧登陆哦!
 H7X7C-GD27C-2PWX7-9B28H-PCGRB

**Reply by MaxMKMax** [reply]
@TGuby 您试试看这个.

YBPGK-VM34J-TRMQ6-4MT8X-4CK28

**Reply by TGuby** [reply]
可以了，非常感谢

**Reply by MaxMKMax** [reply]
@TGuby 嗯嗯好的。不客气。谢谢你对蓝灯的支持!

**Reply by MaxMKMax** [reply]
@TGuby 
如果您不介意的话, 请您到应用商店给我们五星好评! 谢谢哦!

**Comment by SuperF223** [comment]
@Mike777Khoo 你好，专业版用户变成免费版了，请提供激活码,谢谢。账户：icesee233@163.com

**Comment by MaxMKMax** [comment]
@SuperF223 
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/a77a4842-0b09-47f6-b2ed-daed4954b137)

客服说您重新登录就可以了. 用邮件登录.

**Comment by helaowan** [comment]
我的也突然变成免费版，我登录专业版，却说我的激活码出错。然后，我重新购买的蓝灯专业版，用的信箱还是helaowan@126.com，结果钱花出去了，信箱里却没有收到任何信息，也没有新的激活码。我花了两份钱，原来的激活码无效，新买的专业版却没给我发新的激活码，这是怎么回事？我的账户：helaowan@126.com,我2024年1月购买的订单号是：2024011322001360251401427905，我2024年7月新买的订单号：2024070322001360251459110697
